### DISCUSSION TITLE
How to set the pore/throat size in Delaunay networks?

### DISCUSSION THREAD

**Discussion by mourendong** [PMEAL/OpenPNM#2218] [Q&A] [answered]
Hi, everyone
I followed the example "Generate Random Networks Based on Delaunay Triangulations, Voronoi Tessellations, or Both", and created the pore network.
But the pore/throat size always changed every time the program runs.
How to deal with it and make the pore/throat size generally in agreement with the experiment dates?

**Comment by jgostick** [comment, marked as answer]
You can't set the PSD/TSD with these networks.  The sizes are dictated by the local density of the initial points.  So if several points are close together then they will have small sizes, while pores that are isolated in space have room to accomodate a large pore.  You can in principle override the pores sizes by hand (``net['pore.diameter'] = <some sizes>``, but this runs the risk of having pores that overlap, which creates unphysical values for things like throat length and pore volumes (counting some volume twice).  You could potentially control the pore sizes indirectly by controlling how you place the initial points passed to the tessellation, but this is not a function we have in OpenPNM at this time.  Can you describe what you're trying to accomplish, and maybe I can suggest something?
